src/main/java/com/stripe/model/Money.java

```diff
@@ -11,4 +12,44 @@ public Long getAmount() {
 	public String getCurrency() {
 		return currency;
 	}
+
+  public SourceTypes getSourceTypes() {
+    return sourceTypes;
+  }
+
+  public static class SourceTypes extends StripeObject {
+    Long alipayAccount;
+    Long bankAccount;
+    Long bitcoinReceiver;
+    Long card;
+
+    public Long getAlipayAccount() {
```

Comment:
Just in the interest of convention, would you mind adding setters for these as well? (e.g. `setAlipayAccount`) They won't be used in any production paths, but could be useful for testing and such.